packages/sign_in_with_apple/lib/src/widgets/sign_in_with_apple_button.dart

```diff
@@ -20,79 +23,157 @@ class SignInWithAppleButton extends StatelessWidget {
         assert(iconAlignment != null),
         super(key: key);
 
+  /// The callback which will be called when the button is pressed.
```

Comment:
```suggestion
  /// The callback that is be called when the button is pressed.
```

from https://api.flutter.dev/flutter/material/MaterialButton/onPressed.html / https://api.flutter.dev/flutter/cupertino/CupertinoButton/onPressed.html

(See also `builder` which uses a different pattern (but also another use case) https://api.flutter.dev/flutter/widgets/Builder/builder.html)